core/database/src/main/java/com/pronoidsoftware/core/database/RoomLocalAgendaDataSource.kt

```diff
@@ -126,10 +218,16 @@ class RoomLocalAgendaDataSource @Inject constructor(
             .combine(getTasksForDate(targetDate)) { reminders, tasks ->
                 (reminders + tasks).sortedBy { it.startDateTime }
             }
+            .combine(getEventsForDate(targetDate)) { items, events ->
+                (items + events).sortedBy { it.startDateTime }
+            }
```

Comment:
You can also work with a single combine block:
```
return combine(
    getTasksForDate(targetDate),
    getEventsForDate(targetDate),
    getRemindersForDate(targetDate)
) { tasks, events, reminders -> 
    (tasks + events + reminders).sortedBy { it.startDateTime }
}
```

Reply:
ah, much better, thanks!